File: codeforces/low_difficulty/difficulty_800/87.brain's_photos_[707A]/brain's_photos.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int n, m, t;
  char p;
  bool is_blackandwhite;
  cin >> n >> m;
  t = n * m;
  is_blackandwhite = true;
  while (t--) {
    cin >> p;
    if (p != 'B' && p != 'W' && p != 'G') {
      is_blackandwhite = false;
    }
  }
  if (is_blackandwhite) {
    cout << "#Black&White" << endl;
  }
  else {
    cout << "#Color" << endl;
  }
  return 0;
}

File: codeforces/low_difficulty/difficulty_800/87.brain's_photos_[707A]/brain's_photos.py
# Author: José Rodolfo (jric2002)
n, m = [int(e) for e in input().split(" ")]
is_blackandwhite = True
pxs = []
while (n):
  pxs.extend(input().split(" "))
  n -= 1
for p in pxs:
  if (p != 'B' and p != 'W' and p != 'G'):
    is_blackandwhite = False
if (is_blackandwhite):
  print("#Black&White")
else:
  print("#Color")